components/filter/filter.js

```diff
@@ -415,9 +419,13 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 						?hidden="${item.hidden}"
 						key="${item.key}"
 						label="${item.text}"
+						number="${item.count}"
```

Comment:
This looks a little odd. Are you sure you meant to include this?

Reply:
It isn't meant to be there, have taken it off.